tests/sentry/api/endpoints/test_project_rules_configuration.py

```diff
@@ -213,3 +214,37 @@ def test_high_priority_issue_condition_feature(self):
                 "sentry.rules.conditions.high_priority_issue.ExistingHighPriorityIssueCondition"
                 in [filter["id"] for filter in response.data["conditions"]]
             )
+
+    def test_has_in_feature(self):
+        with self.feature({"organizations:issues-alerts-is-in": False}):
+            response = self.get_success_response(self.organization.slug, self.project.slug)
+            tagged_event_filter = next(
+                (
+                    filter
+                    for filter in response.data["filters"]
+                    if filter["id"] == "sentry.rules.filters.tagged_event.TaggedEventFilter"
+                ),
+                None,
+            )
+            assert tagged_event_filter
+            filter_list = [
+                choice[0] for choice in tagged_event_filter["formFields"]["match"]["choices"]
+            ]
+            assert MatchType.IS_IN not in filter_list
+
+        # NOTE: failing because the False case above is removing the choice from the registry :think:
```

Comment:
For some reason this is failing because it looks like the logic is modifying a cached response 🤔 

trying to figure out why this is...